src/standalone-analytics.ts

```diff
@@ -61,19 +61,20 @@ function getWriteKey(): string | undefined {
   return writeKey
 }
 
-export function install(): Promise<void> {
+export async function install(): Promise<void> {
   const writeKey = getWriteKey()
+  const options = window.analytics?._loadOptions ?? {}
   if (!writeKey) {
     console.error(
       'Failed to load Write Key. Make sure to use the latest version of the Segment snippet, which can be found in your source settings.'
     )
     return Promise.resolve()
   }
 
-  return AnalyticsBrowser.standalone(
+  const standaloneAnalytics = (await AnalyticsBrowser.standalone(
```

Comment:
looks way cleaner ✨✨

Reply:
Does it make sense to assign directly to window.analytics and skip creating the `standaloneAnalytics` variable?

Reply:
@chrisradek Makes sense!

Reply:
@chrisradek Seem kind of a smell  that AnalyticsBrowser.standalone returns an analytics type that we have to make a type assertion about? Is there an objection to me refactoring that?

Reply:
No objection - not sure how worth it is though :smile: `AnalyticsBrowser.standalone` uses the correct return type, it's just we internally declare the global analytics object to be what you'd get from the Segment source snippet.

If you have really strong feelings about it you can probably just remove the Symbol.iterator from the `StandaloneAnalytics` type, clean up the unused types and everything _should_ just work then.